packages/react-native-jigsaw/src/components/RadioButton/RadioButtonRow.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from "react";
+import {
+  StyleProp,
+  ViewStyle,
+  StyleSheet,
+  TouchableOpacity,
+  TextStyle,
+} from "react-native";
+import RadioButton, {
+  SEED_DATA as RADIO_BUTTON_SEED_DATA,
+  RadioButtonProps,
+} from "./RadioButton";
+import { View } from "react-native";
+import Text from "../Text";
+import { useRadioButtonGroupContext } from "./context";
+import {
+  createTextProp,
+  createTextStyle,
+  FORM_TYPES,
+} from "../../core/component-types";
+
+export interface RadioButtonRowProps extends Omit<RadioButtonProps, "onPress"> {
+  label: string | React.ReactNode;
+  value: string;
+  labelContainerStyle: StyleProp<ViewStyle>;
+  radioButtonStyle?: StyleProp<ViewStyle>;
+  labelStyle?: StyleProp<TextStyle>;
+  onPress?: (value: string) => void;
+  direction?: "row" | "row-reverse";
```

Comment:
instead of making this a string, can you make this an enum? Row | RowReverse?

you can make it so the enum translates to a lowercase row or row-reverse